pallets/drand/src/lib.rs

```diff
@@ -600,16 +452,24 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
+	fn get_chain_hash() -> &'static str {
+		if cfg!(not(feature = "mainnet")) {
+			return QUICKNET_CHAIN_HASH;
+		}
+		
+		MAINNET_CHAIN_HASH
+	}
+
```

Comment:
This does not scale well. I would suggest having an enum with the supported versions of drand `quicknet` and `mainnet` so far. And pass the value we want to use in the pallet configuration.